woocommerce/wc-admin
# Component: add external d3chart package

This PR attempts to move the chart component and d3chart component into the @woocommerce/components package. It incorporates the update from #961 

### Screenshots
![screenshot 2018-11-28 15 07 04](https://user-images.githubusercontent.com/1160732/49153764-51fef800-f31f-11e8-93f0-f0d852630167.png)

### Detailed test instructions:
- navigate to `WooCommerce > Analytics > Products` report
- select a date range with known orders
- toggle the chart between intervals and type (bar or line)
- hover over legend items and select/deselect items
- hover over the chart to ensure the tooltip renders as expected
- tab through the page to ensure the chart accessibility features respond as it expected
